### ISSUE TITLE
miraculous.to

### ISSUE THREAD

**Issue by rihaq** [bogachenko/fuckfuckadblock#375]
### Basic information

**Check the link:**  `https://miraculous.to/en/`
**Category:** Breakage
**Reported from:**  US

### User environment

**Browser:**  Kiwi browser
**Adblocker:**  uBlock Origin dev
**Extensions:**  Violentmonkey, FastForward, Libredirect
**DNS:**  N/A

### Additional message

 Can't click anything in homepage. Unclickable. I think they are trying to block adblocker

**Comment by bogachenko** [comment]
> Can't click anything in homepage.

What exactly can't you click on this page and how is it related to my filter, **what does it break**?

>Newsflash
>Upcoming Episodes

Not clickable in principle, there is no link to the page, these are banners.

**Comment by bogachenko** [comment]
I can't fix what I can't reproduce, what isn't there.

I installed Kiwi Browser (although in fact it was not necessary, since this problem did not recur on Chromium either), installed the uBO dev extension and with the settings and filters (+ FFA list) by default I went through your site, the result - everything works .***
Disable other extensions, [especially Violentmonkey](https://github.com/bogachenko/fuckfuckadblock/wiki/Problematic-extensions), the problem is somewhere on your side.

Did you do [it](https://github.com/bogachenko/fuckfuckadblock/wiki/Support-policy#before-sending-a-report)?
